partner_document/wizard/res_partner_expired_document_wizard.py

```diff
@@ -0,0 +1,42 @@
+from datetime import date
+
+from odoo import fields, models
+
+
+class ResPartnerExpiredDocumentWizard(models.TransientModel):
+    _name = "res.partner.expired.document.wizard"
```

Comment:
The _name should be "partner.document.expired.wzard" and modify the class and views and everything else accordingly